.github/ISSUE_TEMPLATE/external-core-team.yml

```
@@ -0,0 +1,49 @@
+name: dbt Core - Request changes to docs
+description: File a docs update request that is not already tracked in Orch team's Release Plans (Notion database).
+title: "[Core] <title>"
+labels: ["content","dbt Core"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening a dbt Core documentation issue! Here are a few things to consider:
+          * You can submit ideas or suggest changes to our content by opening an [Issue](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose).
+          * Before filing an issue, please [search our current issues](https://github.com/dbt-labs/docs.getdbt.com/issues) to avoid duplicates.
+          * Please read the [Contributing guide](https://github.com/dbt-labs/docs.getdbt.com#contributing) if you want to open a pull request. 
```

sugg priority label if needed
```suggestion
          * Please read the [Contributing guide](https://github.com/dbt-labs/docs.getdbt.com#contributing) if you want to open a pull request. 
          * Please add the priority label (Optional)
```

---

Just changed to add the label name.